extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -56,6 +57,12 @@ function init(
 	context.set('options.trackingOptIn', isOptedIn);
 	adProductsUtils.setupNpaContext();
 
+	const stickySlotsLines = legacyContext.get('opts.stickySlotsLines');
+	if (stickySlotsLines && stickySlotsLines.length) {
+		context.set('templates.stickyAd.lineItemIds', stickySlotsLines);
+		context.push('slots.TOP_LEADERBOARD.defaultTemplates', 'stickyAd');
```

Comment:
Be aware that this won't work on AE2 mobile-wiki, but we know about that and this is fine :>